app/policies/proposal_policy.rb

```diff
@@ -98,7 +98,7 @@ def pending_delegate?
 
   def pending_approval!
     check(self.pending_approver? || self.pending_delegate?,
-          "A response has already been logged a response for this proposal")
+          "A response has already been logged for this proposal")
```

Comment:
would it be possible to do something like`"This proposal has already been approved by #{approver.email_address}"` ? are there situations other than an already approved proposal where someone would see this error msg?